### ISSUE TITLE
importing rp master breaks

### ISSUE THREAD

**Issue by vivek-bala** [radical-cybertools/radical.pilot#1415]
Using RP master, I tried to import radical pilot and I get a versioning error. Same for 'radical-stack' command.

```
(test-rp) vivek@vivek-HP-Pavilion-m6-Notebook-PC:~/Research/repos/radical.pilot/examples$ python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import radical.pilot as rp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vivek/ves/test-rp/local/lib/python2.7/site-packages/radical/pilot/__init__.py", line 10, in <module>
    import radical.utils as _ru
  File "/home/vivek/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/__init__.py", line 61, in <module>
    version_branch, sdist_name, sdist_path = get_version()
  File "/home/vivek/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/get_version.py", line 92, in get_version
    raise RuntimeError ("Cannot determine version from %s" % paths)
RuntimeError: Cannot determine version from ['/home/vivek/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/..', '/home/vivek/ves/test-rp/local/lib/python2.7/site-packages/radical/utils']
>>> 

```

This maybe due to a mismatch with the numbering scheme for either rp, saga or ru.

**Comment by andre-merzky** [comment]
Can you manually check what versions you have installed for the three layers?  Thanks

**Comment by vivek-bala** [comment]
Sure! I gave it another go. I installed from pypi. 

RP version: 0.46
Saga version: 0.46
RU version: 0.46

See installation log below:

```
$ pip install radical.pilot
Collecting radical.pilot
  Downloading radical.pilot-0.46.tar.gz (10.6MB)
    100% |████████████████████████████████| 10.6MB 140kB/s 
Collecting saga-python>=0.44 (from radical.pilot)
  Downloading saga-python-0.46.tar.gz (697kB)
    100% |████████████████████████████████| 706kB 1.2MB/s 
Collecting radical.utils>=0.44 (from radical.pilot)
  Downloading radical.utils-0.46.tar.gz (128kB)
    100% |████████████████████████████████| 133kB 2.3MB/s 
Collecting pymongo==2.8 (from radical.pilot)
Collecting python-hostlist (from radical.pilot)
Collecting netifaces==0.10.4 (from radical.pilot)
Collecting setproctitle (from radical.pilot)
Collecting ntplib (from radical.pilot)
Collecting msgpack-python (from radical.pilot)
Collecting pyzmq (from radical.pilot)
  Using cached pyzmq-16.0.2-cp27-cp27mu-manylinux1_x86_64.whl
Collecting apache-libcloud (from saga-python>=0.44->radical.pilot)
  Using cached apache_libcloud-2.1.0-py2.py3-none-any.whl
Collecting parse (from saga-python>=0.44->radical.pilot)
Collecting future (from radical.utils>=0.44->radical.pilot)
Collecting colorama (from radical.utils>=0.44->radical.pilot)
  Using cached colorama-0.3.9-py2.py3-none-any.whl
Collecting requests (from apache-libcloud->saga-python>=0.44->radical.pilot)
  Using cached requests-2.18.4-py2.py3-none-any.whl
Collecting urllib3<1.23,>=1.21.1 (from requests->apache-libcloud->saga-python>=0.44->radical.pilot)
  Using cached urllib3-1.22-py2.py3-none-any.whl
Collecting idna<2.7,>=2.5 (from requests->apache-libcloud->saga-python>=0.44->radical.pilot)
  Using cached idna-2.6-py2.py3-none-any.whl
Collecting chardet<3.1.0,>=3.0.2 (from requests->apache-libcloud->saga-python>=0.44->radical.pilot)
  Using cached chardet-3.0.4-py2.py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests->apache-libcloud->saga-python>=0.44->radical.pilot)
  Using cached certifi-2017.7.27.1-py2.py3-none-any.whl
Building wheels for collected packages: radical.pilot, saga-python, radical.utils
  Running setup.py bdist_wheel for radical.pilot ... done
  Stored in directory: /home/vivek91/.cache/pip/wheels/a4/68/ff/694bf1de7f4edb0bcc22c4db12b60572eb583ee73706b476e5
  Running setup.py bdist_wheel for saga-python ... done
  Stored in directory: /home/vivek91/.cache/pip/wheels/58/cd/d8/c31d75478dc946806b57f786e20ffab3f5cd39882aa8232da4
  Running setup.py bdist_wheel for radical.utils ... done
  Stored in directory: /home/vivek91/.cache/pip/wheels/a6/81/b3/0339ef112b96edf83ebeb666066b715f8c0bd1e87c6178a085
Successfully built radical.pilot saga-python radical.utils
Installing collected packages: urllib3, idna, chardet, certifi, requests, apache-libcloud, future, colorama, netifaces, radical.utils, parse, saga-python, pymongo, python-hostlist, setproctitle, ntplib, msgpack-python, pyzmq, radical.pilot
Successfully installed apache-libcloud-2.1.0 certifi-2017.7.27.1 chardet-3.0.4 colorama-0.3.9 future-0.16.0 idna-2.6 msgpack-python-0.4.8 netifaces-0.10.4 ntplib-0.3.3 parse-1.8.2 pymongo-2.8 python-hostlist-1.17 pyzmq-16.0.2 radical.pilot-0.46 radical.utils-0.46 requests-2.18.4 saga-python-0.46 setproctitle-1.1.10 urllib3-1.22
```

Reproducing error after above installation:

```
(test-rp) vivek91@js-17-212:~$ radical-stack
Traceback (most recent call last):
  File "/home/vivek91/ves/test-rp/bin/radical-stack", line 3, in <module>
    import radical.utils as ru
  File "/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/__init__.py", line 61, in <module>
    version_branch, sdist_name, sdist_path = get_version()
  File "/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/get_version.py", line 92, in get_version
    raise RuntimeError ("Cannot determine version from %s" % paths)
RuntimeError: Cannot determine version from ['/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/..', '/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils']

$ python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import radical.pilot as rp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/pilot/__init__.py", line 10, in <module>
    import radical.utils as _ru
  File "/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/__init__.py", line 61, in <module>
    version_branch, sdist_name, sdist_path = get_version()
  File "/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/get_version.py", line 92, in get_version
    raise RuntimeError ("Cannot determine version from %s" % paths)
RuntimeError: Cannot determine version from ['/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/..', '/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils']
```

**Comment by vivek-bala** [comment]
Wait! I think this is something new in pip! There is a 'local' folder inside the ve, see the path:
```
/home/vivek91/ves/test-rp/local/lib/python2.7/site-packages/radical/utils/..
```

I expected (it used to be):
```
/home/vivek91/ves/test-rp/lib/python2.7/site-packages/radical/utils/..
```

If I check inside the local folder, they are all soft links:
```
(test-rp) vivek91@js-17-212:~/ves/test-rp/local$ pwd
/home/vivek91/ves/test-rp/local
(test-rp) vivek91@js-17-212:~/ves/test-rp/local$ ll
total 8
drwxrwxr-x 2 vivek91 vivek91 4096 Aug 22 10:52 ./
drwxrwxr-x 7 vivek91 vivek91 4096 Aug 22 10:52 ../
lrwxrwxrwx 1 vivek91 vivek91   29 Aug 22 10:52 bin -> /home/vivek91/ves/test-rp/bin/
lrwxrwxrwx 1 vivek91 vivek91   33 Aug 22 10:52 include -> /home/vivek91/ves/test-rp/include/
lrwxrwxrwx 1 vivek91 vivek91   29 Aug 22 10:52 lib -> /home/vivek91/ves/test-rp/lib/
```

**Comment by vivek-bala** [comment]
Versions being used:

```
vivek91@js-17-212:~/ves/test-rp$ virtualenv --version
15.0.1
vivek91@js-17-212:~/ves/test-rp$ pip --version
pip 8.1.1 from /usr/lib/python2.7/dist-packages (python 2.7)
```

**Comment by andre-merzky** [comment]
This was caused by an apparent change on the interpretation of `__file__`, which broke the mechanism to look for the correct `VERSION` file.  This is now fixed, and I pushed a hotfix release (0.46.1 for RP, RS, 0.46.2 for RU).

Please give it a try and let me know if that resolves the issue for you!

**Comment by vivek-bala** [comment]
Fixed for me.
